src/App.vue

```diff
@@ -204,43 +394,134 @@ export default {
         // },
 
         favorites(favorites) {
-            localStorage.setItem('favorites', JSON.stringify(favorites))
+            localStorage.setItem('favorites', JSON.stringify(favorites));
 
-            return favorites
+            return favorites;
         },
     },
 
     methods: {
+        addColumn () {
+            this.inputColumn.push({id:(this.index++), name:'', type: ''});
+        },
+
+        removeRow (row) {
+            this.inputColumn.splice(row, 1);
+        },
+
+        toggleModal (title, add, type) {
+            this.showDialogCreate = !this.showDialogCreate;
+            this.dialogTitle = title;
+            this.showAddInput = add;
+            this.dataType = type;
+        },
+
+        closeModal () {
+            this.showDialogCreate = false;
+            this.newName.name = null;
+        },
+
+        capitalize (word) {
+            return word.charAt(0).toUpperCase() + word.slice(1);
+        },
+
+        getElementsColumns (inputColumn) {
+            let columnName=[], columnType=[], elements=[];
+
+            this.inputColumn.forEach(input => {
+                columnName.push( this.capitalize( (input.name).toLowerCase() ) );
+                columnType.push( (input.type).toUpperCase() );
+            })
+
+            for (let i = 0; i < columnName.length; i++) {
+                if (i != columnName.length -1) {
+                    elements += columnName[i] + " " + columnType[i] + ", ";
+                } else {
+                    elements += columnName[i] + " " + columnType[i];
+                }
+            }
+            return elements;
+        },
+
+        executeQuery() {
+            let action, type, query;
+
+            action = this.showAddInput ? 'CREATE ' : 'DROP ';
+            type = (this.dataType == 'database') ? 'DATABASE ' : 'TABLE ';
+
+            // PARA ADICIONAR DATABASE E TABLE
+            if (this.newName.name) {
+
+                if (this.dataType == "table") {
+                    query = action + type + this.newName.name + " ( " + this.getElementsColumns(this.inputColumn) + " );";
+                    this.sendQuery(query, ' table created!', 'Table already exists');
+                }
+                else {
+                    query = action + type + this.newName.name + ";";
+                    this.sendQuery(query, ' database created!', 'Database already exists');
+                }
```

Comment:
Single quote everything... You used this validation 3 times that I counted, it's worth moving to a computed property like `dataTypeTable`